demo/client/backend/mock/README.md

```diff
@@ -0,0 +1,35 @@
+# Start with go chaincode
```

Comment:
it is useful a reference here to the configuration of the debug-print features -- presumably, users need to tool to debug the chaincode.

Reply:
OK, I added a logging section about that in the readme and will fill it with content later :)

Reply:
compiling echo chaincode with `SGX_MODE=DEBUG` enabled led to the following logging output:

```
./mock
2019-12-13 12:29:55.679 UTC [ecc_enclave] Create -> INFO 001 Enclave created with 30069066039298
2019-12-13 12:29:55.680 UTC [ecc_enclave] Bind -> INFO 002 No report pk provided! Call bind with NULL
2019-12-13 12:29:55.681 UTC [ecc_enclave] golog -> INFO 003 WARNING  (/project/src/github.com/hyperledger-labs/fabric-private-chaincode/ecc_enclave/enclave/enclave.cpp:42) [Enclave] Start without TLCC!!!!
2019-12-13 12:29:55.681 UTC [ecc_enclave] golog -> INFO 004 DEBUG  (/project/src/github.com/hyperledger-labs/fabric-private-chaincode/examples/echo/echo_cc.cpp:19) [Enclave] EchoCC: +++ Executing chaincode init +++
2019-12-13 12:29:55.681 UTC [ecc_enclave] golog -> INFO 005 DEBUG  (/project/src/github.com/hyperledger-labs/fabric-private-chaincode/examples/echo/echo_cc.cpp:30) [Enclave] EchoCC: Args: My Auction
2019-12-13 12:29:55.681 UTC [ecc_enclave] golog -> INFO 006 DEBUG  (/project/src/github.com/hyperledger-labs/fabric-private-chaincode/examples/echo/echo_cc.cpp:33) [Enclave] EchoCC: +++ Initialization done +++
[GIN-debug] [WARNING] Creating an Engine instance with the Logger and Recovery middleware already attached.

[GIN-debug] [WARNING] Running in "debug" mode. Switch to "release" mode in production.
 - using env:	export GIN_MODE=release
 - using code:	gin.SetMode(gin.ReleaseMode)

[GIN-debug] GET    /api/getRegisteredUsers   --> main.getAllUsers (4 handlers)
[GIN-debug] GET    /api/clock_auction/getDefaultAuction --> main.getDefaultAuction (4 handlers)
[GIN-debug] POST   /api/cc/invoke            --> main.invoke (4 handlers)
[GIN-debug] POST   /api/cc/query             --> main.invoke (4 handlers)
[GIN-debug] Listening and serving HTTP on :3000
2019-12-13 12:29:57.425 UTC [ecc_enclave] golog -> INFO 007 DEBUG  (/project/src/github.com/hyperledger-labs/fabric-private-chaincode/examples/echo/echo_cc.cpp:41) [Enclave] EchoCC: +++ Executing chaincode invocation +++
2019-12-13 12:29:57.427 UTC [ecc_enclave] golog -> INFO 008 DEBUG  (/project/src/github.com/hyperledger-labs/fabric-private-chaincode/examples/echo/echo_cc.cpp:52) [Enclave] EchoCC: Args: someChaincodeFunction, arg1, arg2
2019-12-13 12:29:57.427 UTC [ecc_enclave] golog -> INFO 009 DEBUG  (/project/src/github.com/hyperledger-labs/fabric-private-chaincode/examples/echo/echo_cc.cpp:70) [Enclave] EchoCC: Response: someChaincodeFunction
2019-12-13 12:29:57.427 UTC [ecc_enclave] golog -> INFO 00a DEBUG  (/project/src/github.com/hyperledger-labs/fabric-private-chaincode/examples/echo/echo_cc.cpp:72) [Enclave] EchoCC: +++ Executing done +++
[GIN] 2019/12/13 - 12:29:57 | 200 |      8.8124ms |      172.17.0.1 | POST     /api/cc/invoke
```

Reply:
In particular, it helped to also do a `make clean && make` inside the the echo folder after setting `SGX_MODE=DEBUG`

Reply:
seems `FABRIC_LOGGING_SPEC` doesn't work with the mock server.

Reply:
Solved:
* setting `DO_DEBUG=true` does the trick
* `SGX_BUILD=DEBUG` triggers the trick
* ... my docker container did not have SGX_BUILD